src/chromium/crPage.ts

```diff
@@ -429,7 +437,7 @@ class FrameSession {
       promises.push(this._client.send('Page.setBypassCSP', { enabled: true }));
     if (options.ignoreHTTPSErrors)
       promises.push(this._client.send('Security.setIgnoreCertificateErrors', { ignore: true }));
-    if (this._isMainFrame() && options.viewport)
```

Comment:
Let's make sure that we don't run updateViewport in case there is no viewport?

Reply:
Done.